lib/Frontend/ArgsToFrontendOptionsConverter.cpp

```diff
@@ -498,6 +503,40 @@ bool ArgsToFrontendOptionsConverter::setUpImmediateArgs() {
   return false;
 }
 
+bool ArgsToFrontendOptionsConverter::computeModuleAliases() {
+  auto list = Args.getAllArgValues(options::OPT_module_alias);
+  if (!list.empty()) {
```

Comment:
Nit: If the list is empty, the loop will run zero times. Will that have the same effect as skipping the loop?

Reply:
Leaving this check as is since I'm allocating a lambda.